libs/orchestration/terraform-azurerm-amido-monitoring/examples/http-logs-metrics/main.tf

```diff
@@ -0,0 +1,51 @@
+data "azurerm_client_config" "current" {}
+
+variable "location" {
+  default = "UK South"
+}
+
+resource "azurerm_application_insights" "monitoring" {
+  name                = "appinsights"
+  location            = var.location
+  resource_group_name = azurerm_resource_group.monitoring.name
+  application_type    = "web"
+}
+
+resource "azurerm_resource_group" "monitoring" {
+  name     = "aidan-test-monitoring-resources"
+  location = var.location
+}
+
```

Comment:
for this it might be better to use a data lookup for these things as they already exist and you won't be able to validate any results this way unless you create a sample application as well to populate data in these app insights resources

Reply:
I figured it would be better to have the example completely stand alone, similar to what we see in the TF documentation but this way works too, updated